lib/Request.php

```diff
@@ -83,6 +83,7 @@ private function getPasteId()
     {
```

Comment:
Ah and BTW (also to catch such errors), in the future, we could use type hints:
```php
private function getPasteId(): string
```

private function getPasteId()

Maybe let's create an issue to tackle this (as it is a general code-style/code improvement one may make)?

Reply:
What do you think, could not you add the type dec. directly here? After we have now technically also ensured it is a string it would be a good starting point, and should be compatible?

Aka as indicated:
```php
private function getPasteId(): string
```

Reply:
I don't like to be inconsistent - if we start doing these, we should do them all and in one go or we have an unreadable mess at our hands. I'm willing to start this experiment in the unit tests, but could not get myself to do it on just one method, so I did all methods on that class.

Result: One docblock was incorrect (return type was actually string not array, probably copy/pasted from prior function) and one return type I can't express with type hints: `void|string` - seems PHP is fine calling this string, even when in some cases no value is returned.